packages/types/src/span.ts

```diff
@@ -133,6 +135,12 @@ export interface Span extends SpanContext {
     spanContext?: Pick<SpanContext, Exclude<keyof SpanContext, 'spanId' | 'sampled' | 'traceId' | 'parentSpanId'>>,
   ): Span;
 
+  /**
+   * Callback to retrieve the root Span (Transaction) in case there is one.
+   * @param callback Will only be invoked in case there is an active transaction
+   */
+  getTransaction(callback: (transaction: Transaction) => void): void;
```

Comment:
Needs to be updated to be aligned with the actual implementation.

Reply:
This can go!